lib/constants.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  COMPONENT_OPTIONS_BLOCK: 'script',
```

Comment:
In any `<page>.vue` file, it will always be a `script` element.

Why create a constant?